### ISSUE TITLE
Remove PuppetJs/dom-bind-notifier

### ISSUE THREAD

**Issue by warpech** [Juicy/dom-bind-notifier#11]
@tomalec can the fork https://github.com/PuppetJs/dom-bind-notifier simply be removed? It is behind https://github.com/Juicy/dom-bind-notifier. I don't know what was the purpose of having this fork.

**Comment by tomalec** [comment]
Yes it can be removed. It was an experiment to use Polymer api more imperatively, but we have solved it with PuppetJs/puppet-polymer-client
